timeless/roles/models.py

```diff
@@ -1,6 +1,16 @@
 """File for models in roles module"""
 
 from timeless import DB
+from enum import Enum
+from sqlalchemy_utils.types.choice import ChoiceType
+
+
+class RoleType(Enum):
+    """ Types of roles """
+    Master = 1
+    Intern = 2
+    Director = 3
+    Manager = 4
```

Comment:
@ammaratef45 can you order them a bit differently? Something like: 

```
Director = 1
Manager = 2
Master = 3
Intern = 4
```

As to convey their actual hyerachical order :D